app/views/shared/_two_columns.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="flex h-screen"
-    data-controller="click-toggle turbo-frame-history"
```

Comment:
This history hack stopped working with turbo-rails 2. But fortunately the `data-turbo-action="advance"` mostly works now in turbo-rails 2 so rather than continuing to support this hack, I've moved us to use `advance`.